### PR TITLE
Created test for Adapter names

### PR DESCRIPTION
**Pull request by silaslenihan**
## Description
Adds a simple test that ensures the source names are formatted properly

## Changes
Adds a test to `packages/core/legos` that tests the names of adapters.

## Quality Assurance

- [ ] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `<ADAPTER_PACKAGE>/schemas/env.json` and `<ADAPTER_PACKAGE>/README.md`
- [ ] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `infra-k8s` configuration file.
- [ ] The branch naming follows git flow (`feature/x`, `chore/x`, `release/x`, `hotfix/x`, `fix/x`) or is created from Clubhouse/Shortcut
- [x] This is related to a maximum of one Clubhouse/Shortcut story or GitHub issue
- [x] Types are safe (avoid TypeScript/TSLint features like any and disable, instead use more specific types)

### CODE REVIEW

**Review by austinborn — COMMENTED** [review]
Thank you for contributing! Would you mind moving this to `.../legos/test/unit/legos.test.ts`?

File: packages/core/legos/test/legos.test.ts
Comment (silaslenihan) [reply packages/core/legos/test/legos.test.ts:6]: @bryanjowers and @justinkaseman perhaps can give more insight as I have just rejoined the team and don't have access to clubhouse yet, but there is a clubhouse card called "EA Legos unit test for names" which says that every adapter name should be tested for correct formatting (ie: no dashes), which is the purpose of this PR. Not entirely sure whether there are other criteria in determining a correctly formatted EA name, but I am waiting on a response from Justin on that one.

Comment (austinborn) [reply packages/core/legos/test/legos.test.ts]: Got it, I can link this to that ticket and add the ticket #.

**Review by austinborn — APPROVED** [review]

**Review by austinborn — CHANGES_REQUESTED** [review]
Can you also make sure the unit tests pass before merging?

File: packages/core/legos/test/legos.test.ts
Comment (silaslenihan) [reply packages/core/legos/test/legos.test.ts]: Thanks!

**Review by justinkaseman — APPROVED** [review]
LGTM.

The changes we talked about will go into another PR and ticket.

**Comment by boxhock** [comment]
@austinborn Mind re-approving and merging this PR?

**Review by austinborn — APPROVED** [review]

**Pull request merged**
